### DISCUSSION TITLE
Add Rack filter to Device Components list

### DISCUSSION THREAD

**Discussion by stef-liesmons** [netbox-community/netbox#10188] [Ideas]
It would be nice to have a rack filter in the device components lists. 
This would be very usefull get a list of all the patch front ports with their labels for a certain rack.

Real life situation:
Go to a rack, check all the patch front ports in one view (without having to figure out out which label belongs to which patch panel).

Desirable Solution:
Show the all Front ports list and filter for the Rack name.

**Comment by jeremystretch** [comment]
For some reason we have region, site group, site, and location filters, but not one for rack. Adding one makes sense to me. Want to open a [feature request](https://github.com/netbox-community/netbox/issues/new/choose) for it?
